calico-enterprise/multicluster/federation/kubeconfig.mdx

```diff
@@ -5,78 +5,47 @@ description: Configure a local cluster to pull endpoint data from a remote clust
-- [Create RemoteClusterConfiguration](#create-remoteclusterconfigurations)
-- [Validate federation and multi-cluster networking](#validate-federation-and-multi-cluster-networking)
-- [Create remote-identity-aware network policy](#create-remote-identity-aware-network-policy)
-- [Troubleshoot](#troubleshoot)
-- [Configure IP pool resources](#configure-ip-pool-resources)
+#### Base requirements for $[prodname] multi-cluster networking:
+- All nodes in the cluster mesh must be able to establish connections to each other via their private IP.
+- All nodes in the cluster mesh must have unique node names.
+- Pod CIDRs between clusters must not overlap.
+- All clusters must have at least one overlay network in common (VXLAN and/or WireGuard).
+- All clusters must have the same `routeSource` setting on `FelixConfiguration`.
 
-### Ensure pod IP routability
-Federation of workload endpoint identities requires [Pod IP routability](./overview#pod-ip-routability) between clusters. If your clusters are using a supported overlay networking mode, $[prodname] can automatically meet this requirement when clusters are connected.
+#### Overlay-specific requirements for $[prodname] multi-cluster networking
+<Tabs>
+<TabItem value="VXLAN" label="VXLAN">
+- The `vxlan*` settings on `FelixConfiguration` must be the same across clusters in the mesh.
+- The underlying network must allow traffic on `vxlanPort` between clusters in the mesh.
+- All clusters must use Calico CNI.
+</TabItem>
+<TabItem value="Wireguard" label="Wireguard">
+- The `wireguard*` settings on `FelixConfiguration` must be the same across clusters in the mesh.
+- The underlying network must allow traffic on `wireguardListeningPort` between clusters in the mesh.
+- All clusters must use Calico CNI OR All clusters must use non-Calico CNI (mixing non-Calico CNI types is supported)
+</TabItem>
+</Tabs>
 
-#### $[prodname] multi-cluster networking
-$[prodname] can automatically extend the overlay networking in your clusters to establish pod IP routes across clusters and thus meet the requirement for Pod IP routability. Only VXLAN overlay is supported at this time.
 
-Ensure the following requirements are met if utilizing $[prodname] multi-cluster networking to achieve pod IP routability:
-- All nodes in the cluster mesh must be able to establish connections to each other via their private IP, and must have unique node names.
-- VXLAN must be enabled on participating IP pools in all clusters, and these IP pool CIDRs must not overlap.
-- `routeSource` and `vxlan*` FelixConfiguration values must be aligned across clusters, and traffic on the `vxlanPort` must be allowed between nodes in the cluster mesh.
-- RemoteClusterConfigurations must be established in both directions for cluster pairs in the cluster mesh.
-- CNI must be Calico.
+## Setup
 
-With these requirements met, multi-cluster networking will be automatically established when RemoteClusterConfigurations are created.
+### Create kubeconfig files
```

Comment:
Suggestion: Make the title describe the user goal, rather than the means of achieving it. 

So, perhaps something more like: Generate credentials for cross-cluster authentication